pkg/uhttp/gocache.go

```diff
@@ -0,0 +1,170 @@
+package uhttp
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"net/http/httputil"
+	"time"
+
+	bigcache "github.com/allegro/bigcache/v3"
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
+	"go.uber.org/zap"
+)
+
+type GoCache struct {
+	ttl         time.Duration
```

Comment:
This value is never read, so I think we can delete it.

Reply:
Fixed